### ISSUE TITLE
[Task]: anomalib训练过程无法正常进行验证

### ISSUE THREAD

**Issue by chenmo2** [openvinotoolkit/anomalib#1361]
### What is the motivation for this task?

我使用最新的anomalib源码配置好参数进行训练，但是验证过程无法实现
Epoch 0:  78%|███████▊  | 21/27 [00:34<00:09,  1.66s/it, loss=nan]
Validation: 0it [00:00, ?it/s]2023-09-26 15:01:04,917 - anomalib.models.padim.lightning_model - INFO - Aggregating the embedding extracted from the training set.
2023-09-26 15:01:05,106 - anomalib.models.padim.lightning_model - INFO - Fitting a Gaussian to the embedding collected from the training set.

Validation:   0%|          | 0/6 [00:00<?, ?it/s]
Validation DataLoader 0:   0%|          | 0/6 [00:00<?, ?it/s]
Epoch 0:  81%|████████▏ | 22/27 [01:26<00:19,  3.94s/it, loss=nan]

### Describe the solution you'd like

如何解决这个验证无法正常运行的问题，验证过程跳过也没有反回具体原因

**Comment by samet-akcay** [comment]
@chenmo2, can you clarify what the problem is? I would suggest using the issue template.
